src/components/Article.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { getArticleByID } from "../utils/api.js";
+
+const Article = ({ article_id }) => {
+  const [article, setArticle] = useState({});
+
+  useEffect(() => {
+    getArticleByID(article_id).then((article) => {
+      setArticle(article);
+    });
+  }, []);
+
+  return (
+    <article>
+      <h2 id="article-title">{article.title}</h2>
```

Comment:
This could be a lot DRY-er if you de-structure the values from article above the return.